orchagent/dash/dashtunnelorch.cpp

```diff
@@ -172,14 +187,17 @@ void DashTunnelOrch::doTask(ConsumerBase &consumer)
                 }
                 else
                 {
+                    result = DASH_RESULT_FAILURE;
```

Comment:
Keeping the tunnel in the consumer here indicates a success, we need to keep it in the consumer to allow for post-bulk ops in `addTunnelMembersPost`. I think a simple `bool` is not able to accurately represent all possible results here which I need to fix it a future PR, but I think at this point we should only consider the task a failure if `addTunnelPost` signals to remove from the consumer AND there is more than one tunnel endpoint.